gpkit/nomials/math.py

```diff
@@ -445,11 +445,10 @@ def _simplify_posy_ineq(self, hmap, pmap=None, allow_tautological=True):
             const_idx = hmap.keys().index(empty_exp)
             self.const_mmap = self.pmap.pop(const_idx)  # pylint: disable=attribute-defined-outside-init
             self.const_coeff = coeff  # pylint: disable=attribute-defined-outside-init
-        if (allow_tautological and
-                (coeff >= -1.*self.feastol or np.isnan(coeff))
+        if (allow_tautological and (coeff >= -self.feastol or np.isnan(coeff))
                 and len(hmap) == 1):  # a tautological monomial!
-            return None  # ValueError("tautological constraint: %s" % self)
-        elif coeff <= -1.*self.feastol:
+            return None  # was ValueError("tautological constraint: %s" % self)
+        elif coeff <= -self.feastol:
```

Comment:
It's funny, when I did that there were some errors, that's why I did -1*. But if this is passing tests good!

Reply:
yeah I added a pylint exemption above this line!

Reply:
Oh ok, I see now! Thanks, will keep that in mind in the future!